app/views/duplicate_logs/_duplicate_log_check_answers.erb

```diff
@@ -7,31 +7,38 @@
 
           <% row.value do %>
             <%= simple_format(
-              get_answer_label(question, @log),
+              get_answer_label(question, log),
               wrapper_tag: "span",
               class: "govuk-!-margin-right-4",
             ) %>
 
-            <% extra_value = question.get_extra_check_answer_value(@log) %>
+            <% extra_value = question.get_extra_check_answer_value(log) %>
 
-            <% if extra_value && question.answer_label(@log, current_user).present? %>
+            <% if extra_value && question.answer_label(log, current_user).present? %>
               <%= simple_format(
                 extra_value,
                 wrapper_tag: "span",
                 class: "govuk-!-font-weight-regular app-!-colour-muted",
               ) %>
             <% end %>
 
-            <% question.get_inferred_answers(@log).each do |inferred_answer| %>
+            <% question.get_inferred_answers(log).each do |inferred_answer| %>
               <span class="govuk-!-font-weight-regular app-!-colour-muted"><%= inferred_answer %></span>
             <% end %>
           <% end %>
-
-          <% row.action(
-            text: question.action_text(@log),
-            href: duplicate_logs_action_href(@log, question.page.id, @original_log_id),
-            visually_hidden_text: question.check_answer_label.to_s.downcase,
-          ) %>
+          <% if @all_duplicates.count > 1 %>
```

Comment:
we could use `many?` here I think if you like, up to you
https://apidock.com/rails/Enumerable/many%3F

Reply:
there's another place in this PR we could as well though it's not a line you've changed